packages/marketing-modal/src/BlobSVG/styles.ts

```diff
@@ -0,0 +1,12 @@
+import { css } from '@leafygreen-ui/emotion';
+import { palette } from '@leafygreen-ui/palette';
+
+export const blobBaseStyles = css`
+  position: absolute;
+  z-index: -1;
+`;
+
+export const blobThemeStyles = (darkMode: boolean) =>
```

Comment:
Did we decide that we want to use objects here instead? cc @spark33

Reply:
I think you're correct